lib/math.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var R = require('ramda');
+
+var operators = {
+  '+': R.add,
+  '-': R.subtract,
+  '*': R.multiply,
+  '/': R.divide,
+  '%': R.modulo,
```

Comment:
> mathMod requires Integer arguments, and returns NaN when the modulus is zero or negative.

I think it's fine to leave it as-is. I don't have an opinion on whether the behavioral difference would be seen as a benefit for this kind of usage.